docs/source/howto/run_codes.rst

```diff
@@ -0,0 +1,376 @@
+.. _how-to:run-codes:
+
+*************************
+How to run external codes
+*************************
+
+This how-to walks you through the steps of setting up a (possibly remote) compute resource, setting up a code on that computer and submitting a calculation through AiiDA (similar to the :ref:`introductory tutorial <tutorial:basic:calcjob>`, but in more detail).
+
+To run an external code with AiiDA, you need an appropriate :ref:`calculation plugin <topics:plugins>`.
+In the following, we assume that a plugin for your code is already available from the `aiida plugin registry <https://aiidateam.github.io/aiida-registry/>`_ and installed on your machine, e.g. using ``pip install aiida-quantumespresso``.
+If a plugin for your code is not yet available, see :ref:`how-to:plugin-codes`.
+
+Throughout the process you will be prompted for information on the computer and code.
+In these prompts:
+
+ * Type ``?`` followed by ``<enter>`` to get help on what is being asked at any prompt.
+ * Press ``<CTRL>+C`` at any moment to abort the setup process.
+   Your AiiDA database will remain unmodified.
+
+.. note::
+
+  The ``verdi`` commands use ``readline`` extensions to provide default answers, which require an advanced terminal.
+  Use a standard terminal -- terminals embedded in some text editors (such as ``emacs``) have been known to cause problems.
+
+.. _how-to:run-codes:computer:
+
+How to set up a computer
+========================
+
+A |Computer| in AiiDA denotes a computational resource on which you will run your calculations.
+It can either be:
+
+ 1. the machine where AiiDA is installed or
+ 2. any machine that is accessible via `SSH <https://en.wikipedia.org/wiki/Secure_Shell>`_ from the machine where AiiDA is installed.
+
+The second option allows managing multiple remote compute resources (including HPC clusters and cloud services) from the same AiiDA installation and moving computational jobs between them.
+
+.. note::
+
+    The second option requires access through an SSH keypair.
+    If your compute resource demands two-factor authentication, you may need to install AiiDA directly on the compute resource instead.
+
+
+Computer requirements
+---------------------
+
+Compute resources that can be used by AiiDA to run calculation come in two types:
+
+1. The computer on which AiiDA itself is installed, i.e., the `localhost`
+2. Remote computers that can be accessed over SSH (possibly :ref:`via a proxy server<how-to:ssh:proxy>`)
```

Comment:
@chrisjsewell Can I suggest to simply merge this with the list above?
I.e. just move the parenthesis to line 34 and get rid of this list here.

Reply:
Good point, I overlooked that list when I reviewed it.

Reply:
see cd5bda7cc2dda0ffbc0eef8f4dbad20c18325d2f